gateway/Dockerfile

```
@@ -0,0 +1,22 @@
+FROM amazoncorretto:17-alpine AS builder
+
+WORKDIR /app
+
+COPY gradlew .
+COPY gradle gradle
+COPY settings.gradle .
+COPY auth-shared auth-shared
+COPY gateway gateway
+
+RUN chmod +x gradlew
+RUN ./gradlew :gateway:clean :gateway:build -x test --no-daemon
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)